_health-care/_js/components/personal-information/NameAndGeneralInfoSection.jsx

```diff
@@ -37,9 +37,7 @@ class NameAndGeneralInfoSection extends React.Component {
               <h4>Date of Birth</h4>
               <span className="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
               <DateInput
-                  day={this.props.data.dateOfBirth.day}
-                  month={this.props.data.dateOfBirth.month}
-                  year={this.props.data.dateOfBirth.year}
+                  value={this.props.data.dateOfBirth}
```

Comment:
1) to be consistent with how we are handling other nested object components, 2) because I thought it would avoid the console errors about required props. Can definitely revert these changes if you think that's best.